src/pkjs/index.js

```diff
@@ -3,7 +3,7 @@ var weather = require('./weather');
 
 var CONFIG_VERSION = 8;
 // var BASE_CONFIG_URL = 'http://localhost:4000/';
-var BASE_CONFIG_URL = 'http://freakified.github.io/TimeStylePebble/';
+var BASE_CONFIG_URL = 'https://udxs.me/TimeStylePebble/';
```

Comment:
Can we keep this as Github pages? Any changes you made you can push to the gh-pages branch.